.github/workflows/e2e-test.yml

```
@@ -0,0 +1,25 @@
+name: e2e Test
+on:
+  push:
+    branches:
+      - development
+      - main
+  pull_request:
+    branches:
+      - development
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn build
+      - run: yarn test:ci
```

```suggestion
      - run: |
           yarn install
           yarn build
           yarn test:ci
```

nitpick, but I find this way nicer

---

True, thanks!